lib/index.d.ts

```diff
@@ -111,6 +111,17 @@ declare namespace Realm {
          * @returns Results<T>
          */
         linkingObjects<T>(objectType: string, property: string): Results<T>;
+
+        /**
+         * @returns string
+         */
+        _objectId(): string;
```

Comment:
are we going to document this method. If it is going to be public lets rename it `objectId`. same for the other methods

Reply:
I think we should keep the underscore and not document it.

Reply:
Are there good reasons this is not just going public right away?

Reply:
The only current demand for this is from internal users. I expect we'll make this public once core supports stable IDs and the functions become applicable to all Realms, rather than only synced ones.